src/util/daos_vol_err.h

```diff
@@ -22,7 +22,7 @@ extern hid_t dv_async_err_g;
 
 #define DAOS_VOL_ERR_CLS_NAME "DAOS VOL"
 #define DAOS_VOL_ERR_LIB_NAME "DAOS VOL"
-#define DAOS_VOL_ERR_VER      "1.0.0"
```

Comment:
Thanks for the fix. We should try to pull this in at build time instead eventually.